tensorflow_datasets/core/registered.py

```diff
@@ -23,6 +23,7 @@
 import abc
```

Comment:
Please add `# Lint as...` and a docstring with instructions to run the script.

Reply:
It already have lint as, and is you want me to include docstring for metadata_cleanup.py here ?

Reply:
My bad. I was looking at the wrong file.